homeassistant/components/weatherkit/coordinator.py

```diff
@@ -37,7 +37,7 @@ def __init__(
             hass=hass,
             logger=LOGGER,
             name=DOMAIN,
-            update_interval=timedelta(minutes=15),
+            update_interval=timedelta(minutes=5),
```

Comment:
This should happen in another PR:

 - its not needed for the lib bump
 - its not visible in the changelog when it comes in via a lib bump

Reply:
Sure, will split this into a different PR

Reply:
Moved to https://github.com/home-assistant/core/pull/105094